### PR TITLE
Update How-secure-is-portugal.md

### PR DESCRIPTION
**Pull request by Ambg05**

### CODE REVIEW

File: manuscript/How-secure-is-portugal.md [hunk 1]
```diff
@@ -14,7 +14,11 @@ The question is, how secure are they? How safe are they? By 'secure', I mean how
 
 ### Our security model is based on lack of attackers
 
-Portugal is safe because there are not enough attackers targeting the current insecurities of the system. This will probably remain the case for the next couple of years. The problem is what happens after that, when the criminals improve their business models and start to focus on Portuguese assets.
+I use the internet, and I use my credit card online, even though I know I might be hacked by a criminal.
+
+I know what the underbelly looks like, so if anybody should be paranoid, it should be me. Despite this, I feel pretty safe from a cyber attack.
```
Comment (DinisCruz) [inline manuscript/How-secure-is-portugal.md]: These two phases are not working and move away from the topic of this paragraph.

I think it is better to remove them

File: manuscript/How-secure-is-portugal.md [hunk 2]
```diff
@@ -58,6 +61,19 @@ The real danger is from criminals who use the internet and vulnerable code for f
 
 We need to make the Portuguese network hostile to this type of criminal activity. The police and hackers (i.e. civil society) have an important role to play in this campaign.
 
+### Attackers' ROI (return on investment)
+
+We must remember that it doesn't take a lot of money to compromise a country or a company. This is the cost of the zero day, this is the worst case scenario to give (add table from slide 26 of C-days presentation).
+
```
Comment (DinisCruz) [inline manuscript/How-secure-is-portugal.md:66]: Add Image and reference to it 

Do you know how to do this?

Btw, are you Mac or Windows? (Mac has a nice screenshot tool)

File: manuscript/How-secure-is-portugal.md [hunk 3]
```diff
@@ -58,6 +61,19 @@ The real danger is from criminals who use the internet and vulnerable code for f
...
+If I had $100,000 to invest, what would I do? I would buy a number of zero-days, some compromised machines inside the Portuguese networks, and I would buy botnets to attack Portuguese companies.
+
+How much money would I make? If I can make a million pounds, it makes business sense, because my only concern is money.
+
+If there is a million dollars to be made from attacking a particular company, somebody will be able to invest $100,000 in the exercise.
+
```
Comment (DinisCruz) [inline manuscript/How-secure-is-portugal.md:72]: Add note about how that has a good RoI (Return on Investment)

File: manuscript/How-secure-is-portugal.md [hunk 4]
```diff
@@ -58,6 +61,19 @@ The real danger is from criminals who use the internet and vulnerable code for f
...
+The good news for us is that the attackers are making far more money doing other things, but they are growing, and what I describe will happen.
+
+What is the return on investment, and who can survive this kind of compromise? My view is that very few companies, especially in Portugal, would survive.
```
Comment (DinisCruz) [inline manuscript/How-secure-is-portugal.md]: remove 'What is the return on investment, and'

File: manuscript/How-secure-is-portugal.md [hunk 5]
```diff
@@ -67,29 +83,28 @@ Once you know about the attack, you will fix it, the security team will get a mu
...
 
 ### Hit by the crossfire
 
-The important question is _"Do we want to do something about this, or be hit by the cross-fire?"_ If we do not consider this question, and address the huge skills shortage in cyber/app security that exists in Portugal, we will pay a heavy price.
+The important question is _"Do we want to do something about this, or be hit by the cross-fire?"_ If we do not consider this question, and address the huge, worldwide, skills shortage in cyber/app security, we will pay a heavy price.
 
-As attackers become more sophisticated, they will gravitate to countries and companies with weaker defenses, since these take longer to ramp up their security when an attack occurs.
+As attackers become more sophisticated, they will gravitate to countries and companies with weaker defenses, since these take longer to ramp up their security when an attack occurs. Acting in a strategic way now presents Portugal, and countries like Portugal, with tremendous opportunities.
```
Comment (DinisCruz) [inline manuscript/How-secure-is-portugal.md]: add note about it also being an insurance policy to what is coming up next

File: manuscript/How-secure-is-portugal.md (hunk 1 above)
Comment (Ambg05) [reply manuscript/How-secure-is-portugal.md]: Done.

File: manuscript/How-secure-is-portugal.md (hunk 2 above)
Comment (Ambg05) [reply manuscript/How-secure-is-portugal.md:66]: Done, but check that I've done it properly!  I'm Windows at the moment, hoping to get a Mac in the New Year.

File: manuscript/How-secure-is-portugal.md (hunk 3 above)
Comment (Ambg05) [reply manuscript/How-secure-is-portugal.md:72]: Done.

File: manuscript/How-secure-is-portugal.md (hunk 5 above)
Comment (Ambg05) [reply manuscript/How-secure-is-portugal.md]: Done.

File: manuscript/How-secure-is-portugal.md (hunk 4 above)
Comment (Ambg05) [reply manuscript/How-secure-is-portugal.md]: Done.

**Pull request merged**